extensions/binding/org.eclipse.smarthome.binding.ntp.test/src/test/java/org/eclipse/smarthome/binding/ntp/test/NtpOSGiTest.java

```diff
@@ -159,7 +171,8 @@ public void testStringChannelTimeZoneUpdate() {
 
         Configuration configuration = new Configuration();
         configuration.put(NtpBindingConstants.PROPERTY_TIMEZONE, TEST_TIME_ZONE_ID);
-
+        configuration.put(NtpBindingConstants.PROPERTY_NTP_SERVER_HOST, TEST_HOSTNAME);
+        configuration.put(NtpBindingConstants.PROPERTY_NTP_SERVER_PORT, TEST_PORT);
```

Comment:
If you have to pass these 2 properties to the `initialize` method in every test method, how about setting them **inside** the `initialize` method and remove the duplicate code from the test methods? Because the server and port won't change for any testmethod, right?